tests/test_paramseq.py

```diff
@@ -325,3 +326,15 @@ def test_parametrized_before_eom_mode(mod_device):
 
     # Just check that building works
     seq.build(amp=3.0)
+
+
+def test_iterable_variable_check():
+    seq = Sequence(reg, device)
+    seq.declare_channel("ch0", "rydberg_global")
+    #
+    delay_t = np.array(100)
+    # A 0-D array is identified as an Iterable but then fails iteration
+    # Giving this as an argument to a sequence call should still work
+    assert isinstance(delay_t, Iterable)
+    seq.delay(delay_t, "ch0")
```

Comment:
It works because `delay` has the decorator `store` that calls the function `verify_parametrization` that call `verify_variable`, which we want to test.